templates/build/small-card-detail.build.hbs

```diff
@@ -52,13 +52,22 @@
       \{{#each entryDetails}}
         <!-- Image template -->
         \{{#ifCond type '==' 'image'}}
-          \{{#if content}}
-            \{{#if labelEnabled}}
-            <div class="small-card-list-detail-label">\{{label}}</div>
-            \{{/if}}
-
-            <div class="small-card-list-detail-body-text">
-              <img src="\{{validateImage content}}">
+          \{{#if (isSingle content)}}
+            \{{#each content}}
+              \{{#if labelEnabled}}
+                <div class="small-card-list-detail-label">\{{label}}</div>
+              \{{/if}}
+              <div class="small-card-list-detail-body-text">
+                <img src="\{{validateImage this}}">
+              </div>
+            \{{/each}}
+          \{{else}}
+            <div class="small-card-multiple-images-holder">
+              \{{#each content}}
+                <div class="small-card-list-multiple-images-item" id="small-card-photoswipe">
```

Comment:
This suggests there will be multiple DOM elements on the same with the same ID. We can't have that.